### PR TITLE
Added a link to the Projects Section 

### PR DESCRIPTION
**Pull request by ujdcodr**
Any projects taken up within the @WebClub-NITK/members  can be added in the projects,html file

### CODE REVIEW

File: projects.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <h2><a id="ongoing-projects" class="anchor" href="#ongoing-projects" aria-hidden="true"><span class="octicon octicon-link"></span></a>Ongoing Projects</h2>
+
+            <!-- List yourself here! Feel free to link to your twitter, facebook, blog, etc. -->
+            <h3> DHCP Relay Enhancement for ns-3 </h3>
+
+            <ul>
+	      <li>Project: Enabling Support for relays in DHCP as an enhancement to ns-3</li>
+	      <li>Students: GVS Tushaar, Ujjwal Pasupulety</li>
+              <li>Mentor: <a href="https://adeepkit01.github.io">Ankit Deepak</a> </li>
+	      <li>Abstract: Improving on our Mentor's implementation of DHCP in ns-3, the aim is to add the enhancement of including relays in a given network. While the basic form of DHCP works with only a client-server model, in the real world, there are relays between the two in large networks that act as mediators between the client and server in assigning new IP addresses and relinquishing unused ones back into the address pool. Our goal is to implement a real world simulation of DHCP in ns-3</li>
```
Comment (adeepkit01) [inline projects.html]: Improving on current implementation of DHCP in ns-3, the aim is to add the enhancement of including relays in a given network.  As for the rest of the statement, I hope you used credible source

File: projects.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	      <li>Project Code: <a href="https://github.com/ujdcodr/dhcprelay">DHCP Relay Enhancement</a>
```
Comment (adeepkit01) [inline projects.html]: DHCP Relay implementation in ns-3

**Review by adeepkit01 — CHANGES_REQUESTED** [review]
I have mentioned cosmetic changes in commits

File: projects.html (hunk 1 above)
Comment (ujdcodr) [reply projects.html]: Is the rest of it fine? I kind of used my own understanding of DHCP to write that

Comment (adeepkit01) [reply projects.html]: It is advantageous to use the
   Dynamic Host Configuration Protocol (DHCP) as defined in RFC 2131 to
   assign customer premise host IP addresses in this environment.
   However, a number of security and scaling problems arise with such
   "public" DHCP use.   RFC 3046 proposed relay-agent option that extends the set of DHCP options as
   defined in RFC 2132.

   The new option is called the Relay Agent Information option and is
   inserted by the DHCP relay agent when forwarding client-originated
   DHCP packets to a DHCP server.  Servers recognizing the Relay Agent
   Information option may use the information to implement IP address or
   other parameter assignment policies.  The DHCP Server echoes the
   option back verbatim to the relay agent in server-to-client replies,
   and the relay agent strips the option before forwarding the reply to
   the client.

Comment (adeepkit01) [reply projects.html]: Use that to keep the introduction inline with RFC

**Comment by adeepkit01** [comment]
Expand my outdated comments for more changes

**Review by adeepkit01 — APPROVED** [review]

**Comment by adeepkit01** [comment]
Merged

**Pull request merged**